docs/source/executing-operations/defer-support.mdx

```diff
@@ -0,0 +1,170 @@
+
+The Apollo Router supports deferring the following fields in your schema:
+
+* Fields that are nested anywhere within an [entity type](#entity-fields)
+* Root fields of the [`Query` type](#fields-of-query)
+
+In most supergraphs, this represents a large percentage of all fields in the schema. [See an example of unsupported fields.](#unsupported-field-example)
+
+### Entity fields
+
+The Apollo Router supports using `@defer` with fields and nested subfields of [entity types](/federation/entities/) in your federated supergraph. Entities are object types with at least one `@key`, which specifies the fields that can uniquely identify any given instance of that type.
+
+Here's an example subgraph schema that defines a `User` entity:
+
+```graphql title="Users subgraph"
+type User @key(fields: "id") {
+  id: ID!
+  name: Name!
+  friends: [User!]!
+}
+
+type Name {
+  firstName: String!
+  lastName: String!
+}
+
+type Query {
+  person(id: ID!): User
+}
+```
+
+Because the `User` type is an entity, a client query can apply `@defer` to any fragment that's nested somewhere within that type (at any depth).
+
+Here are two example queries:
+
+```graphql
+query QueryA($personId: ID!) {
+  person(id: $personId) {
+    id
+    # highlight-start
+    ... on User @defer {
+      name {
+        firstName
+        lastName
+      }
+    }
+    # highlight-end
+  }
+}
+
+query QueryB($personId: ID!) {
+  person(id: $personId) {
+    id
+    name {
+      firstName
+      # highlight-start
+      ... on Name @defer {
+        lastName
+      }
+      # highlight-end
+    }
+  }
+}
+```
+
+### Fields of `Query`
+
+The Apollo Router also supports using `@defer` with root fields of your supergraph's `Query` type:
+
+```graphql
+query PartiallyDeferredQuery {
+  products {
+    id
+  }
+  # highlight-start
+  ... on Query @defer {
+    users {
+      id
+    }
+  }
+  # highlight-end
+}
+```
+
+You _cannot_ use `@defer` with root fields of the `Mutation` type.
+
+### Unsupported field example
+
+Let's say our router's supergraph schema includes the following:
+
+```graphql
+type Book {
+  title: String!
+  author: String!
+}
+
+type Query {
+  books: [Book!]!
+}
+```
+
+The Apollo Router _cannot_ defer part of the following query as requested:
+
+<p style="margin-bottom: 0">⚠️</p>
+
+```graphql
+query DeferredAuthorQuery {
+  books {
+    title
+    # highlight-start
+    ... on Book @defer {
+      author # Can't be deferred
+    }
+    # highlight-end
+  }
+}
+```
+
+This is because `Book.author` is _not_ nested within an entity type. If the `Book` type were an entity, the router _would_ be able to defer this field.
+
+**If a query attempts to defer an unsupported field:**
+
+* The router waits for the unsupported field to finish resolving before it sends its initial response, just like it does for all non-deferred fields.
```

Comment:
This is sentence is largely true, but as mentioned in my bigger remark at the beginning of this section for details, I feel the definition of what an "unsupported field" is a bit vague (in particular, it's not so much field that are unsupported per se, but "field in a particular position within the @defer").

Reply:
I've eliminated this particular language and "unsupported field" in general in favor of "non-deferred field"